src/VisualStudio/CSharp/Impl/ProjectSystemShim/CSharpProjectShim.ICSInputSet.cs

```diff
@@ -102,17 +102,17 @@ public void SetWin32Icon(string iconFileName)
 
         public void SetFileAlignment(uint align)
         {
-            // This option is used only during emit. Since we no longer use our in-proc workspace to emit, we can ignore this value.
+            SetOption(ref _fileAlignment, align);
```

Comment:
Inlcude a comment on this.  That even though we don't do in-proc emit, we still want to thread this down so that people can properly query for it and know what the emit options actually are.